src/styles/messages.scss

```diff
@@ -533,6 +610,11 @@
 		@include scrollbar;
 	}
 
+	&__item {
+		margin: 12px;
```

Comment:
the space between messages is too big. i think margin: 0 5px,padding: 4px is better